cmd/client.go

```diff
@@ -5,20 +5,27 @@ import (
 	"time"
 
 	shieldv1beta1 "github.com/odpf/shield/proto/v1beta1"
+	"github.com/spf13/cobra"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
 )
 
 func createConnection(ctx context.Context, host string) (*grpc.ClientConn, error) {
 	opts := []grpc.DialOption{
-		grpc.WithInsecure(),
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
 		grpc.WithBlock(),
 	}
 
 	return grpc.DialContext(ctx, host, opts...)
 }
 
-func createClient(ctx context.Context, host string) (shieldv1beta1.ShieldServiceClient, func(), error) {
-	dialTimeoutCtx, dialCancel := context.WithTimeout(ctx, time.Second*2)
+func createClient(cmd *cobra.Command) (shieldv1beta1.ShieldServiceClient, func(), error) {
```

Comment:
@mabdh Any specific reason to pass the entire `cmd` object instead of just the host for this method?

Reply:
This is because we use cmd flag as the source of truth of cli config. So we are copying data from cli.Config if exist and not the other way around. 

Passing cmd is just to avoid having one line `cmd.Flags().GetString("host")` for every client that needs it.

Reply:
Now I am thinking again, I think the ideal way should be making the `cli.Config` as the source of truth if there is any flag passed, we override the fields in the `cli.Config` itself and use the `cli.Config` in our logic.

Reply:
+1

Reply:
+1. it is possible to bind cobra and viper together to do this automatically (assuming cli.Config is being loaded from that). But it requires some work. 

(alternate option might be to just have a helper function `readFlags(cmd *cobra.Command, into *cli.Config)` which checks if flags are set and overrides the struct fields may be)

Reply:
@spy16 @ravisuhag updated the code, can you guys please check, basically does  similar flow like the `readFlags` that @spy16 suggested above.